src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,893 @@
+or `aminoacylation reaction <https://www.ebi.ac.uk/ols/ontologies/mi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMI_1143>`_.
+
+
+Therefore, other vocabularies like the `Gene Ontology (GO) <https://www.ebi.ac.uk/QuickGO/>`_ or the
+`Molecular Process Ontology (MOP) <https://www.ebi.ac.uk/ols/ontologies/mop>`_ were used to find corresponding
+interaction terms. These terms were then annotated with the name, namespace and identifier. The following tables shows
+examples of how the interactions from IntAct were mapped to BEL.
+
++-------------+-------------+----------------------------------------------------------------------+-------------------------------------------------------------+
+| Source Type | Target Type | Interaction Type                                                     | BEL Example                                                 |
++=============+=============+======================================================================+=============================================================+
+| Protein     | Protein     | psi-mi:"MI:0193"(amidation reaction)                                 | p('uniprot', 'P62865') increases p('uniprot', 'P10731')     |
++-------------+-------------+----------------------------------------------------------------------+-------------------------------------------------------------+
+| Protein     | Protein     | psi-mi:"MI:1327"(sulfurtransfer reaction)                            | p('uniprot', 'Q46925') increases p('uniprot', 'P0AGF2')     |
++-------------+-------------+----------------------------------------------------------------------+-------------------------------------------------------------+
+| Protein     | Protein     | psi-mi:"MI:0945"(oxidoreductase activity electron transfer reaction) | p('uniprot', 'P0A3E0') increases p('uniprot', 'P21890')     |
++-------------+-------------+----------------------------------------------------------------------+-------------------------------------------------------------+
+| Protein     | Protein     | psi-mi:"MI:0217"(phosphorylation reaction)                           | p('uniprot', 'P53999') increases p('uniprot', 'P68400')     |
++-------------+-------------+----------------------------------------------------------------------+-------------------------------------------------------------+
+| Protein     | Protein     | psi-mi:"MI:0567"(neddylation reaction)                               | p('uniprot', 'Q86XK2') increases p('uniprot', 'Q15843')     |
++-------------+-------------+----------------------------------------------------------------------+-------------------------------------------------------------+
+| Protein     | Protein     | psi-mi:"MI:1148"(ampylation reaction)                                | p('uniprot', 'P60953-2') increases p('uniprot', 'Q9BVA6')   |
```

Comment:
the BEL isn't formatted correctly (should be written as CURIE) and the pmods/activities are missing. TBH i would suggest writing this out all very carefully by hand

Reply:
What do you mean with the BEL is not formatted correctly? I thought this would be the syntax

Reply:
I wrote a function to construct the entire table and consider all the special cases, so this is less prone to errors than writing it out by hand.

Reply:
The Pmods are not missing, they are all included.